test/integration/model/create.test.js

```diff
@@ -984,8 +984,8 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           return user.save({
             logging(sql) {
               test = true;
-              expect(sql).to.contain('ARRAY[]::INTEGER[]');
-              expect(sql).to.contain('ARRAY[]::VARCHAR(255)[]');
+              expect(sql).not.to.contain('ARRAY[]::INTEGER[]');
```

Comment:
These tests should be moved to unit tests, but that is too much work, should be tackled in different PR. Please add some assertion which covers what is expected here, like may be just `ARRAY[]`?

Reply:
I've tidied this up a bit:

* https://github.com/sequelize/sequelize/pull/9492/files#diff-4b224bbc09dfb867f8671d77e298bb8cR955
* https://github.com/sequelize/sequelize/pull/9492/files#diff-4b224bbc09dfb867f8671d77e298bb8cR955